app/web/features/auth/FeaturePreview.tsx

```diff
@@ -30,6 +40,18 @@ export default function FeaturePreview() {
         </Typography>
       </MuiAlert>
       <NotificationSettings className={classes.section} />
+      {isAccountInfoLoading ? (
+        <CircularProgress />
+      ) : accountInfoError ? (
+        <Alert severity="error">{accountInfoError.message}</Alert>
+      ) : (
+        <>
+          <ChangePhone
+            className={classes.section}
+            {...(accountInfo as GetAccountInfoRes.AsObject)}
```

Comment:
```suggestion
            accountInfo={accountInfo}
```
It's more preferred to do pass props explicitly than spreading, as it's quite opaque what props the component accepts otherwise, as shown by the need to type assert...